clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/manifest/KubernetesDeleteManifestOperation.java

```diff
@@ -68,7 +68,9 @@ public OperationResult operate(List<OperationResult> priorOutputs) {
     } else if (options.containsKey("cascading")) {
       deleteOptions.setOrphanDependents(options.get("cascading").equalsIgnoreCase("false"));
     }
-
+    if (options.containsKey("gracePeriodSeconds")) {
+      deleteOptions.setGracePeriodSeconds(Long.parseLong(options.get("gracePeriodSeconds")));
```

Comment:
Small comment: parse the `gracePeriodSeconds` in a try/catch, so the operation doesn't fail if the parse fails or the value is null, and just log a warning in case it fails.